daras_ai_v2/bot_integration_widgets.py

```
@@ -553,6 +559,188 @@ def web_widget_config(bi: BotIntegration, user: AppUser | None, hostname: str |
+        gui.html(
+            '<a class=" btn btn-theme btn-secondary w-100 mb-3" '
+            'href="https://t.me/BotFather" target="_blank">'
+            f"{icons.telegram} Open BotFather"
+            "</a>",
+        )
+        gui.markdown("**Step 2: Name your bot & add a username**")
+        gui.markdown("Send: *`/newbot`* to BotFather to get started")
+        _copiable_field("Bot name", bot_name)
+        _copiable_field("Bot username", bot_username)
+
+        gui.markdown("**Step 3: Paste in your bot’s token**")
```

Ruff (0.15.5)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)